reference/cpp-api.md

```diff
@@ -472,7 +472,8 @@ The following tables describe the C++ classes and their methods.
+| &nbsp;&nbsp; std::string [getTypeName](supervisor.md#wb_supervisor_node_get_type)() const;                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |                                                                                                                                                                   
+| &nbsp;&nbsp; std::string [getId](supervisor.md#wb_supervisor_node_get_def)() const;                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
```

Comment:
Personally I won't add the new `getDef` function in between `getTypeName()` and `getBaseTypeName` but before `getType` or after `getBaseTypeName` in order to keep semantical groups of functions.

The same comment applies to all the APIs.